packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ZoomingScatterPlot/ZoomingScatterPlot.js

```diff
@@ -215,13 +216,19 @@ function ZoomingScatterPlot({
       panning={panning}
       top={topPosition}
       width={width}
+      move={interactionMode === 'move'}
       setOnPan={setOnPan}
       setOnZoom={setOnZoom}
-      zoomingComponent={ScatterPlot}
       zoomConfiguration={zoomConfiguration}
       zooming={zooming}
-      {...scatterPlotProps}
-    />
+    >
+      {(zoomProps) =>(
+        <ScatterPlot
+          {...scatterPlotProps}
+          {...zoomProps}
+        />
```

Comment:
The scatter plot has different focus when dragging with these changes.
Note the focus below the x-axis.
![scatter_plot_interactionMode](https://github.com/user-attachments/assets/8b69102b-9bc2-471a-aad0-052fa3e25e46)
I think maybe the SVGComponents/Background component could use `focusable` and `tabIndex` similar to the changes with `SVGImages`?

Reply:
Good point. I'll try making the background focusable.